Docs/source/building/juwels.rst

```diff
@@ -0,0 +1,64 @@
+.. _building-juwels:
+
+Juwels (JSC)
+============
+
+The `Juwels supercomputer <https://www.fz-juelich.de/ias/jsc/EN/Expertise/Supercomputers/JUWELS/JUWELS_node.html>`_ is located at JSC.
+
+See `this page <https://apps.fz-juelich.de/jsc/hps/juwels/quickintro.html>`__ for a quick introduction.
+
+Installation
+------------
+
+We use the following modules and environments on the system.
+
+.. code-block:: bash
+
+   # please set your project account
+   export proj=<yourProject>
+
+   # required dependencies
+   module load GCC
+   module load OpenMPI
+   module load CUDA
+
+   # AMReX specifics
+   export GPUS_PER_SOCKET=2
+   export GPUS_PER_NODE=4
+
+Note that for now WarpX must rely on OpenMPI instead of the recommended MPI implementation on this platform MVAPICH2.
```

Comment:
Uh no, why?

This will likely cost performance, because system libs properly configure the network stack.

Reply:
I compared 2-node jobs (uniform plasma) on Summit and Juwels (weak-scaled because Juwels only has 4 GPUs/node), and they show similar performances, but the communication routines were ~15% slower on Juwels (that's a very rough estimate), so not totally off. I am not sure AMReX officially supports MVAPICH (besides MPICH and OpenMPI), so I ended up with weird compilation errors where some compiler flags were not properly passed (`nvcc: -pipe is unknown` or sth like that).

Reply:
MVAPICH2 should work if MPICH works, it's based on the latter and adds splendid infiniband support (among other optimizations). Could you try if building WarpX against MVAPICH2 with CMake works?